### PR TITLE
Move Workshop database expiry to Database Tab

### PR DESCRIPTION
**Pull request by LionelColaso**
Moving This since it makes more sense here

Screenshot 2024-07-13 131728

### CODE REVIEW

**Review by hk21702 — CHANGES_REQUESTED** [review]
Seems good. Do me a favor and update from main to get the new settings model, and then reorder the corresponding setting so that it matches the new organization.

**Comment by LionelColaso** [comment]
> Seems good. Do me a favor and update from main to get the new settings model, and then reorder the corresponding setting so that it matches the new organization.

yeah sure , also would it be good idea to add a toogle to enable disable this notification time ?

File: app/controllers/settings_controller.py [hunk 1]
```diff
@@ -422,6 +421,9 @@ def _update_view_from_model(self) -> None:
             self.settings.external_steam_metadata_repo
         )
         self.settings_dialog.steam_workshop_db_github_url.setCursorPosition(0)
+        self.settings_dialog.db_builder_database_expiry.setText(
```
Comment (hk21702) [inline app/controllers/settings_controller.py:424]: this should probably be renamed for clarity

**Review by hk21702 — CHANGES_REQUESTED** [review]

File: app/controllers/settings_controller.py (hunk 1 above)
Comment (LionelColaso) [reply app/controllers/settings_controller.py]: ok done should be good now

**Comment by LionelColaso** [comment]
All should be fine now

**Review by oceancabbage — APPROVED** [review]

**Review by hk21702 — APPROVED** [review]

**Pull request merged**